frontend/src/pages/ViewApplication.tsx

```diff
@@ -2,40 +2,67 @@ import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
 
 import api, { PopulatedReview } from "../api";
+import ApplicationHeader from "../components/ApplicationHeader";
+import ScoreCard from "../components/ScoreCard";
+import StageNotes from "../components/StageNotes";
 import { useAlerts } from "../hooks/alerts";
 import { makeComparator } from "../util";
-import ApplicationView from "../views/ApplicationView";
-
-import { ReviewView } from "./EditReview";
 
 export default function ViewApplication() {
   const { applicationId } = useParams();
   const { alerts, addAlert } = useAlerts();
 
   const [reviews, setReviews] = useState<PopulatedReview[]>([]);
+
   useEffect(() => {
     api
       .getFilteredReviews({ application: applicationId || "" })
       .then(setReviews)
       .catch(addAlert);
   }, [applicationId]);
 
+  const sortedReviews = reviews
+    .slice()
+    .sort(
+      makeComparator((r) => [
+        r.stage.pipelineIdentifier,
+        r.stage.pipelineIndex,
+        r.reviewerEmail || "",
+      ])
+    );
+
+  // Group reviews by stage
+  const groupedReviews = sortedReviews.reduce((acc, review) => {
+    const key = review.stage.name;
+    if (!acc[key]) acc[key] = [];
+    acc[key].push(review);
+    return acc;
+  }, {} as Record<string, PopulatedReview[]>);
+
+  // Get unique stage names
+  const stages = Array.from(new Set(sortedReviews.map((r) => r.stage.name)));
```

Comment:
Can you sort reviews by most -> least recent stage like I did [here](https://github.com/TritonSE/TSE-Fulcrum/blob/main/frontend/src/pages/Home.tsx#L27)? We almost always want to see the reviews from the current stage when we look at the application